gateway/gateway.go

```diff
@@ -565,6 +657,11 @@ func (gateway *HandleT) printStackHandler(w http.ResponseWriter, r *http.Request
 	// stackTrace := string(byteArr[:n])
 	// fmt.Println(stackTrace)
 	w.Write(byteArr)
+
+	// n := runtime.Stack(byteArr, true)
```

Comment:
We need this bottom half. That will trim down the array and also add the nodeID as part of the request.